modules/byo-vpc-multiple-az.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * osd_cluster_create/managing-byo-vpc-multiple-az.adoc
+
+[id="byo-vpc-multiple-az_{context}"]
+= Configuring subnets for a VPC with multiple availability zones
+
+For VPCs with multiple availability zones, you must configure a `public` and `private` subnet per availability zone, and then, create the appropriate routing rule and NAT gateway for each of the availability zones.
+
+.Procedure
+
+. Create six subnets for three separate zones as follows:
++
+[cols="3",options="header"]
+|===
+| Availability zone | Subnet name | CIDR IP address
+| A | `publicA`  | `10.0.0.0/19`
+| A | `privateA` | `10.0.96.0/19`
+| B | `publicB`  | `10.0.32.0/19`
+| B | `privateB` | `10.0.128.0/19`
+| C | `publicC`  | `10.0.64.0/19`
+| C | `privateC` | `10.0.160.0/19`
+|===
++
+[NOTE]
+====
+Each subnet is for a separate zone, each with a `public` and `private` subnet.
+====
+
+. Create a `RouteTable` that has a route table association with the three `public` subnets.
+. Create a routing rule on the `RouteTable` for IP address `0.0.0.0/0` to the internet gateway that was created.
+. Create three NAT gateways in the subnet as follows:
+.. A NAT gateway on `publicA` called `privateA` with an elastic internet address (EIP).
+.. A NAT gateway on `publicB` called `privateB` with an EIP.
+.. A NAT gateway on `publicC` called `privateC` with an EIP.
```

Comment:
In line with `RouteTable` suggestion above:
````
.. A NAT gateway on the `publicA` subnet called `privateA` with ...
````
If that feels a like too much noun soup, consider making these sentences to make it clearer:
````
On the `publicA` subnet, create a NAT gateway with an elastic internet address (EIP). Name this gateway `privateA`.
````